BunqSdk.Tests/README.md

```diff
@@ -34,4 +34,6 @@ same directory, fill in your sandbox user data and rename the copy to `config.js
 
 ## Execution
 To run tests via Rider, you can right click on the `BunqSdk.Tests` solution and should be able to run
-the tests cases form the IDE.
+the tests cases from the IDE.
+To run the tests via Visual Studio, you can use the Test Explorer window. Use default shortcut Ctrl+R, A
```

Comment:
Is there a shortcut for the Mac edition of Visual Studio too?

Reply:
For VS Code there is an [extension] (https://github.com/formulahendry/vscode-dotnet-test-explorer) for testing. Although I use VS Code more and more frequently, I still do 95% of .NET development with Visual Studio, so I don't have much feedback on this extension, yet. I am also not sure if this shortcut will work with VS Code. 

Are you guys using Rider? What is your feedback? (Sorry for the off topic of the PR)

Reply:
Yea I use Rider, please list this extension in the README as well then to ensure that other know that they must install the extension 👍

Reply:
@antao myself I don't code in C# at all, but we use the Jetbrains editors mostly yes (PhpStorm, PyCharm, etc.). But in my case, no Rider.

Reply:
@OGKevin can you check for the shortcut on MacOS then?